api/internal/target/kusttarget_test.go

```diff
@@ -40,7 +40,7 @@ func TestLoad(t *testing.T) {
 			},
 		},
 		"nonsenseLatin": {
-			errContains: "error converting YAML to JSON",
+			errContains: "found a tab character that violates indentation",
```

Comment:
Nit: this points out the test isn't testing what it claimed, i.e. it is testing tab indent vs space indent instead of random text vs yaml. The real error in a nonsense case is fine too (`yaml: line 6: could not find expected ':'`) and also an improvement on the misleading error we gave before!